README.md

```diff
@@ -40,13 +40,16 @@ A service for candidates to apply for initial teacher training.
 - Yarn 1.12.x
 - PostgreSQL 9.6
 - Graphviz 2.22+ (`brew install graphviz`) to generate the [domain model diagram](#domain-model)
+- Redis 5.0.x
 
 ## <a name="dev-prerequisites"></a>Prerequisites for development
 
 `docker` and `docker-compose`
 
 ## <a name="dev-env-setup"></a>Setting up the development environment
 
+If you are not planning to run the server processes in docker:
```

Comment:
?

Reply:
:+1: that sentence doesn't make much sense any longer. Will remove.